storagenode/storagenodedb/database.go

```diff
@@ -653,6 +653,29 @@ func (db *DB) Migration() *migrate.Migration {
 					`INSERT INTO piece_space_used (total) select ifnull(sum(piece_size), 0) from pieceinfo_`,
 				},
 			},
+			{
+				Description: "Add disqualified field to reputation",
+				Version:     18,
+				Action: migrate.SQL{
+					`DROP TABLE reputation;`,
```

Comment:
Why drop and recreate the table rather than adding a single column?

Reply:
I want `updated_at` to be the last column) May be that’s overkill. But this table is not being used at the moment, so recreating it should not be a problem